utils.py

```diff
@@ -453,6 +453,20 @@ def get_time_in_millisecs(datetime_obj):
     return msecs + python_utils.divide(datetime_obj.microsecond, 1000.0)
 
 
+def convert_date_to_datetime(date_obj):
```

Comment:
You might not need this -- see comment above re YYYY-MM-DD being a better representation for date.

Reply:
Yup, ended up not needing it will remove.